src/main/assets/_social.styl

```diff
@@ -89,7 +89,7 @@
 			social(3)
 		&[href*=stackoverflow]
 			social(4)
-		&[href*=forum]
+		&[href*=discuss]
```

Comment:
This had me scratching my head for a bit... strange but I didn't want to redo it now.

Reply:
that's is that actually?

Reply:
hmm, I guess these are the icons on on the left-upper corner. Maybe we should remove the mailing list one.

Reply:
It's applying a style based on a substring match of the URL!

The style is used to set the background image offset to display the correct icon from a sprite image.

Reply:
I think it's good to have a link to the forum there... the icon is unfortunate but we can fix later.